lm_admin_guide.md

```diff
@@ -316,6 +320,57 @@ The only non-Python dependency is **PostgreSQL** (back-end/client), which is
 required by the `psycopg2` Python package.
 
 
+
+## How to upgrade
+
+### Upgrade a Docker deployment
+Existing Dockerized deployments can be easily upgraded to a more recent LifeMonitor version by going through the following steps:
+1. stop `lm` service
+    ```bash
+    docker-compose stop lm
+    ``` 
+2. make a backup of LifeMonitor to a temp path within the `db` container
+    ```bash
+    docker-compose exec db /bin/bash -c "PGPASSWORD=\${POSTGRESQL_PASSWORD} pg_dump -U \${POSTGRESQL_USERNAME} \${POSTGRESQL_DATABASE} > /tmp/lifemonitor_backup.sql"
+    ```
+3. copy backup from the `db` container to a local path
+    ```bash
+    docker cp life_monitor_db_1:/tmp/lifemonitor_backup.sql ${HOME}/lifemonitor_backup.sql
+    ```
+4. teardown all the services in `prod` mode
+    ```bash
+    make down
+    ```
+5. update your local copy of LifeMonitor sources (via `git clone` or `git pull`)
+6. restart all the services 
+    ```bash  
+    make start
+    ```
+    or type `make start-dev` to restart all the services in `dev` mode.
+
+
+As a result, the up-to-date LifeMonitor instance will be started and the existing data will be migrated to the proper database schema. 
+
```

Comment:
How come `flask db upgrade` is not necessary in this case?

Reply:
The upgrade is automatically performed within the `init` container.